index.css

```diff
@@ -0,0 +1,186 @@
+body {
+    background-color: #f2f2f2;
+    color: #333;
+    font-weight: 400;
+    line-height: 1.3;
+    margin: 32px;
+    font-family: Arial, Arial, Helvetica, sans-serif;
+}
+
+img {
+    max-width: 100%;
+}
+
+.nav {
+    box-sizing: border;
+    padding: 15px;
+}
+
+.topnav {
+    list-style-type: none;
+    margin: 0;
+    padding: 0;
+    overflow: hidden;
+    background-color: #333;
+}
+
+.topnav a {
+    float: left;
```

Comment:
I usually reserve floating for elements that need to have text wrapped around them; here, instead, the effect you want is to have side-by-side elements and `display:inline-block` will get you that. You would need to apply this to the `li` instead of the `a` element here.

Also, you have floated an element without a [clearfix](https://mica-web.github.io/learn/css/best-practices#wrap-all-floated-elements-in-another-element-with-the-clearfix-class-applied) applied to the parent.